cookiecutter.yaml

```
@@ -4,7 +4,7 @@ about: The ICESat-2 Hackweek is a 5-day collaborative learning experience design
 apply:
   url: '#'
   title: Applications Closed
-hackweek_mission: https://uwhackweek.github.io/hackweeks-as-a-service/mission.html
+hackweek_mission: https://icesat-2.hackweek.io/mission.html
 banner:
   title: The ICESat-2 Hackweek 2022
```

```suggestion
  title: The ICESat-2 Hackweek
```
(looking at the preview, 2022 is repeated on the next line of the title banner)